src/main/java/com/google/api/codegen/configgen/ProtoPagingParameters.java

```diff
@@ -21,10 +21,11 @@
 public class ProtoPagingParameters implements PagingParameters {
   private static final String PARAMETER_PAGE_TOKEN = "page_token";
   private static final String PARAMETER_NEXT_PAGE_TOKEN = "next_page_token";
-  private static final String PARAMETER_MAX_RESULTS = "page_size";
+  private static final String PARAMETER_PAGE_SIZE = "page_size";
+  private static final String PARAMETER_MAX_RESULTS = "max_results";
 
   private static final ImmutableList<String> IGNORED_PARAMETERS =
-      ImmutableList.of(PARAMETER_PAGE_TOKEN, PARAMETER_MAX_RESULTS);
+      ImmutableList.of(PARAMETER_PAGE_TOKEN, PARAMETER_PAGE_SIZE);
```

Comment:
This logic is used *only* in gapic config generation, i.e. in basically dead code not used anywhere now. We can add it there but it will not affect anything (I just did not want to touch dead code if not necessary).

Reply:
Oh, OK. Right, we're in `configgen`. But you _are_ changing/adding `PARAMETER_PAGE_SIZE` and `PARAMETER_MAX_RESULTS`, so then it's not really dead?

Reply:
This was just a queer coincidence. the java constant for the "page_size" string literal was called `PARAMETER_MAX_RESULTS` argument was. So to add "max_results" parameter and that code to keep making sense I had to rename the old constant to `PARAMETER_PAGE_SIZE` first =)

Reply:
It is dead code but it may be still used in discogapic (i'm not sure if it is, but discogapic depends on gapic yaml generation). I did not want to touch that if not necessary.